index.html

```diff
@@ -29,10 +30,10 @@
     </section>
     <section class="how">
```

Comment:
Is there any way to genercize (not a word) the naming of "how" so that we could decide to use that style over again?  I didn't load the HTML but how similar is it with the what?  Are there reusable components there?   If so then I think we should setup the CSS that way.